packages/scripts/config/webpack.config.js

```diff
@@ -19,6 +19,11 @@ const isProduction = process.env.NODE_ENV === 'production';
 const mode = isProduction ? 'production' : 'development';
 
 const config = {
+	optimization: {
+		// Only concatenate modules in production, when not analyzing bundles.
+		concatenateModules:
+			mode === 'production' && ! process.env.WP_BUNDLE_ANALYZER,
+	},
```

Comment:
Is there a relevant entry we should mention in `scripts/CHANGELOG.md` ?

Reply:
Added an entry 👍